File: src/main/java/com/github/theword/queqiao/tool/constant/CommandConstantMessage.java
package com.github.theword.queqiao.tool.constant;

/**
 * 命令相关的常量消息
 *
 * <p>包含命令执行过程中用于提示或日志输出的静态字符串。
 */
public class CommandConstantMessage {
    public static final String RELOAD_CONFIG = "加载配置文件完成";

    public static final String RECONNECT_MESSAGE = "正在尝试重连 %s 的 WebSocket 客户端...";
    public static final String RECONNECT_NOT_OPEN_CLIENT = "存在未处于打开状态的 Websocket Client...";
    public static final String RECONNECT_ALL_CLIENT = "正在重连所有 Websocket Client...";
    public static final String RECONNECT_NO_CLIENT_NEED_RECONNECT = "没有客户端需要重连";
    public static final String RECONNECTED = "已重新连接";
}

File: src/main/java/com/github/theword/queqiao/tool/constant/CommandConstantMessage.kt
package com.github.theword.queqiao.tool.constant

object CommandConstantMessage {
    const val RELOAD_CONFIG: String = "配置文件重载完成"

    const val RECONNECT_MESSAGE: String = "正在尝试重连至：%s 的 WebSocket 服务器..."
    const val RECONNECT_NOT_OPEN_CLIENT: String = "即将重新连接未处于开启状态的 Websocket Client..."
    const val RECONNECT_ALL_CLIENT: String = "即将重新连接所有的 Websocket Client..."
    const val RECONNECT_NO_CLIENT_NEED_RECONNECT: String = "没有需要重连的 Websocket Client"
    const val RECONNECTED: String = "重连任务已完成"
}
